transformations/use_acronyms/transformation.py

```diff
@@ -0,0 +1,72 @@
+from interfaces.SentenceOperation import SentenceOperation
+from tasks.TaskTypes import TaskType
+
+"""
+Use Acronyms.
+    Change groups of words for their equivalent acronym.
+    The acronyms were scraped from Wikipedia's List of Acronyms and naively filtered.
+"""
+
+
+def transformation(sentence, acronyms):
+    new_sentence = sentence
+    for key in acronyms.keys():
+        if key in new_sentence:
```

Comment:
One suggestion, if we add "key.lower() in new_sentence.lower()" in the if statement, we can also cover cases like "massachusetts institute of technology", "Massachusetts institute of technology" and "massachusetts institute of Technology" etc.

Reply:
I was weary of dealing with casing because of how it might affect the meaning between input and output. For example, a sentence like: "They visited the new York stadium" will end up becoming "They visited the NY stadium" which is not the same. By sticking to the _standard_ capitalization the chance of these miss changes diminishes.

I guess it is a matter of preference as well as how relevant it is to keep the meaning of the original sentence for the specific task for which the data is being augmented.

In any case, I added support for lowercasing as a class parameter.